src/scripts/clipperUI/components/sectionPicker.tsx

```diff
@@ -288,10 +288,9 @@ export class SectionPickerClass extends ComponentBase<SectionPickerState, Sectio
 			}
 		}
 
-		// Display an un-retryable message if API responds with a code that is not retryable
-		// Note that if this is an unhandled api response code, we assume it is not retryable
-		if (this.state.apiResponseCode && !OneNoteApiUtils.isRetryable(this.state.apiResponseCode)) {
-			localizedStrings.notebookLoadFailureMessage = Localization.getLocalizedString("WebClipper.SectionPicker.NotebookLoadUnretryableFailureMessage");
+		// If we can show a better message, especially an actionable one, we do
+		if (this.state.apiResponseCode && OneNoteApiUtils.getLocalizedErrorMessage(this.state.apiResponseCode)) {
```

Comment:
why the removal of the `!isRetryable` check?

without it, it seems like if the API responds with a _retryable_ API response that isn't "actionable", then we're going to fallback to the `NotebookLoadUnretryableFailureMessage`, instead of the correct fallback of `NotebookLoadFailureMessage`. Might be missing something...